ports/esp32/boards/OLIMEX_ESP32_POE/board.json

```diff
@@ -0,0 +1,24 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "BLE",
+        "WiFi",
+        "MicroSD",
+        "Battery Charging",
+        "Ethernet",
+        "PoE",
+        "Breadboard friendly"
+    ],
+    "id": "esp32",
+    "images": [
+        "ESP32-POE-ISO-1.jpg"
+    ],
+    "mcu": "esp32",
+    "product": "ESP32",
```

Comment:
@robert-hh this line should be more descriptive, eg "Olimex ESP32 POE".  The download page will use this as the title.

Reply:
Done